### PR TITLE
Adding Jorge Ruvalcaba

### PR DESCRIPTION
**Pull request by jorgeruvalcaba**
Also removed a duplicated tag I found while scrolling

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -5824,6 +5823,28 @@ module.exports = [
     phone: 'iphone',
     tags: ['Full Stack', 'Developer', 'Front End', 'JavaScript', 'React'],
   },
+  {
+    name: 'Jorge Ruvalcaba',
+    description:
+      'Software Engineer, Senior Frontend Engineer, Taco Master and internet weirdo 🤓',
+    url: 'https://jorgeruvalcaba.dev/uses/',
```
Comment (davidak) [inline src/data.js:5830]: i only get

> Not Found
> You just hit a route that doesn't exist... the sadness.

**Review by davidak — CHANGES_REQUESTED** [review]
LGTM

except that the uses page does not exist... but that is the whole purpose of this

File: src/data.js (hunk 1 above)
Comment (jorgeruvalcaba) [reply src/data.js:5830]: My bad 🤦  . Didn't push the page before but now it's alive. You can check it out [here](https://jorgeruvalcaba.dev/uses/).

**Comment by jorgeruvalcaba** [comment]
@davidak resolved! 😅

**Review by davidak — APPROVED** [review]

**Comment by davidak** [comment]
very good. thank you!

**Pull request merged**